src/main/java/at/hannibal2/skyhanni/features/skillprogress/SkillButtonInInventory.kt

```diff
@@ -0,0 +1,133 @@
+package at.hannibal2.skyhanni.features.skillprogress
+
+import at.hannibal2.skyhanni.api.SkillAPI.customGoalConfig
+import at.hannibal2.skyhanni.api.SkillAPI.overflowConfig
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.events.InventoryCloseEvent
+import at.hannibal2.skyhanni.events.InventoryFullyOpenedEvent
+import io.github.moulberry.notenoughupdates.events.ReplaceItemEvent
+import io.github.moulberry.notenoughupdates.util.Utils
+import net.minecraft.client.player.inventory.ContainerLocalMenu
+import net.minecraft.init.Blocks
+import net.minecraft.item.ItemStack
+import net.minecraftforge.fml.common.eventhandler.EventPriority
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class SkillButtonInInventory {
+
+    private var showItem = false
+
+    private val toggleStackSizeEnabled by lazy {
+        val stack = ItemStack(Blocks.stained_hardened_clay)
+        Utils.createItemStack(
+            stack.item,
+            "§aStack Size Currently ON",
+            13,
+            "§8(From SkyHanni)",
+            "",
+            "§7§eClick to toggle!")
+    }
+
+    private val toggleTooltipEnabled by lazy {
+        val stack = ItemStack(Blocks.stained_hardened_clay)
+        Utils.createItemStack(
+            stack.item,
+            "§aTooltip Currently ON",
+            13,
+            "§8(From SkyHanni)",
+            "",
+            "§7§eClick to toggle!")
+
+    }
+
+    private val toggleCustomGoalEnabled by lazy {
+        val stack = ItemStack(Blocks.stained_hardened_clay)
+        Utils.createItemStack(
+            stack.item,
+            "§aCustom Goal Currently ON",
+            13,
+            "§8(From SkyHanni)",
+            "",
+            "§7§eClick to toggle!")
+
+    }
+
+    private val toggleStackSizeDisabled by lazy {
+        val stack = ItemStack(Blocks.stained_hardened_clay)
+        Utils.createItemStack(
+            stack.item,
+            "§cStack Size Currently OFF",
+            14,
+            "§8(From SkyHanni)",
+            "",
+            "§7§eClick to toggle!")
+    }
+
+    private val toggleTooltipDisabled by lazy {
+        val stack = ItemStack(Blocks.stained_hardened_clay)
+        Utils.createItemStack(
+            stack.item,
+            "§cTooltip Currently OFF",
+            14,
+            "§8(From SkyHanni)",
+            "",
+            "§7§eClick to toggle!")
+    }
+
+    private val toggleCustomGoalDisabled by lazy {
+        val stack = ItemStack(Blocks.stained_hardened_clay)
+        Utils.createItemStack(
+            stack.item,
+            "§cCustom Goal Currently OFF",
+            14,
+            "§8(From SkyHanni)",
+            "",
+            "§7§eClick to toggle!")
+    }
```

Comment:
Could extract these into a fucntion that makes them and just changes the first line